website/content/docs/configure-hcp-packer.mdx

```diff
@@ -0,0 +1,88 @@
+
+You can configure any Packer template to push metadata to the HCP Packer registry, but we recommend that you use HCL2 Packer templates. 
+The HCL2 support in Packer allows you to codify how Packer interacts with the registry and customize your image metadata. 
+
+Users wishing to use HCP Packer with JSON are encouraged to get started with the environment variables and eventually work to migrate from JSON to HCL2 configuration
+templates. 
+
+## HCP Packer Environment Variables
+
+HCP Packer configuration through environment variables is currently the only way to publish artifacts from a Packer JSON template. 
+
+The HCP Packer environment variables are made available for users who wish to publish their artifacts to an active HCP Packer registry
+without making any changes to their existing template. Making it an ideal solution for users looking to get started with an existing template; both JSON and HCL2.
+
+HCP Packer environment variables must be configured before executing a Packer build command. They work with all template types but are required for legacy JSON templates. 
+
+** Required Environment Variables**
+
+- `HCP_CLIENT_ID`         - The HCP client ID of the HashiCorp Cloud Platform service principle that will be authenticating Packer to the HCP Packer registry.
+
+- `HCP_CLIENT_SECRET`    - The HCP client secret of the HashiCorp Cloud Platform service principle that will be authenticating Packer to the HCP Packer registry.
+
+- `HCP_PACKER_BUCKET_NAME` -  The name of the Image bucket where Packer will store information about each artifact from a build within the configured template. 
+
+** Optional Environment Variables**
+
+- `HCP_PACKER_BUILD_FINGERPRINT` -  A unique identifier assigned to a build run to determine if metadata for a template build on the registry is complete. A fingerprint can 
+not be more than 32 characters long.
+
+- `HCP_PACKER_REGISTRY`    -  Allows users to toggle HCP Packer publishing off. Only valid when all required variables have been set. Allowed values are [0|OFF].
+
+~> **Build Fingerprint**: By default, Packer uses the HEAD Git SHA for the template file to create a fingerprint. 
+If your template is not version controlled, you must set the HCP_PACKER_BUILD_FINGERPRINT environment variable and set it to a unique value each time you call packer build. 
+Packer will not build the template unless it detects a build fingerprint.
+
+## HCP Packer Registry HCL Block
+
+HCL2 templates, along with support for environment variables, can be configured using a `hcp_packer_registry` block, lets you customize the metadata Packer sends to HCP Packer Registry. It configures the details of an image that is created or updated within the HCP Packer registry. 
+
+The HCP Packer registry block in HCL is the preferred method for configuring Packer for the HCP Packer registry,
```

Comment:
We've got a lot of detailed information about this and the bucket labels, etc. in the HCP docs that I don't think we should replicate here (but that users may want to know!). I wonder if we can shorten this and just link out to the existing content we have in the HCP docs?

Reply:
Or, maybe just link more quickly to the registry blockpage --> it looks like it has an expanded version of this same content. The thing I want to avoid is giving users JUST enough that they try to do things with this information, but not enough to give them all the context they need to set up a good workflow/ to avoid stubbing their toe.

Reply:
This is a fair concern and one I struggled with. I like the idea of linking sooner to not duplicate and guiding folks to the information they need to build a solid workflow. I would venture to say that the `hcp_packer_block` page should also link out to the complete documentation at https://cloud.hashicorp.com/docs/packer/store-image-metadata/template-configuration. This way we can really hone in on the clear list of requirements in the HCP Packer docs. 

We can do the same for the JSON docs once the HCP Packer docs have been updated with the JSON details.